controllers/topics.controller.js

```diff
@@ -0,0 +1,13 @@
+const fetchTopics = require("../models/fetchTopics.model.js");
+
+function getTopics(req, resp, next) {
+  fetchTopics(req.query)
+    .then((data) => {
+      resp.status(200).send(data.rows);
```

Comment:
Best practise is to send your response back on a relevant key, such as topics, so your response would look like
`{topics: data.rows}`
This could have a bit of a clearer name than `data`, as well, perhaps `topics`?